packages/teams_memory/teams_memory/interfaces/types.py

```diff
@@ -33,6 +33,7 @@ class InternalMessageInput(BaseMessageInput):
     type: ClassVar = "internal"
     deep_link: ClassVar[None] = None
     created_at: datetime = Field(default_factory=datetime.now)
+    source: Optional[str] = None
```

Comment:
new addition?

Reply:
Hmm looks like this was leftover from my previous PR. Removing, sorry about that.